docs/pages/database-access/enroll-google-cloud-databases/spanner.mdx

```diff
@@ -0,0 +1,218 @@
+---
+title: Database Access with Cloud Spanner
+description: How to configure Teleport database access with GCP's Cloud Spanner.
+---
+
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="Cloud Spanner" dbConfigure="with a service account"!)
+
+## How it works
+
+(!docs/pages/includes/database-access/how-it-works/iam.mdx db="Spanner" cloud="Google Cloud"!)
+
+<Tabs>
+
+<TabItem label="Self-Hosted">
+![Teleport Cloud Spanner Self-Hosted](../../../img/database-access/guides/spanner_selfhosted.png)
+</TabItem>
+
+<TabItem label="Teleport Enterprise (cloud-hosted)">
+![Teleport Cloud Spanner Cloud-Hosted](../../../img/database-access/guides/spanner_cloud.png)
+</TabItem>
+
+</Tabs>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- Google Cloud account
+- Google Cloud Spanner database.
+- A host where you will run the Teleport Database Service, e.g. a Compute Engine
+  instance.
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/8. Create a service account for the Teleport Database Service
+
+Teleport uses [IAM authentication](https://cloud.google.com/spanner/docs/iam)
+to access the Cloud Spanner API.
```

Comment:
```suggestion
Teleport uses [IAM authentication](https://cloud.google.com/spanner/docs/iam)
to access the Cloud Spanner API.
```
I think we can remove this detail since we include it in the "How it works" section. If we want to provide more info on IAM authentication, maybe we could make it more explicit that we're linking to the GCP docs in the `## Next steps` section.

Reply:
good point - removed. We link the docs elsewhere already too